content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Deploying Cribl Stream Containers on HPE GreenLake
+date: 2024-01-17T21:25:38.155Z
+author: Elias Alagna, HPE Distinguished Technologist & Kam Amir, Cribl Director
+  of Technical Alliances
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - ezmeral, hpe-ezmeral-container-platform, hpe-ezmeral-data-fabric,
+    kubernetes, hpe-greenlake, as-a-service, PCE, Private Cloud Enterprise,
+    logging, Splunk, GreenLake, Cribl
+---
+Hewlett Packard Enterprise and [Cribl](https://cribl.io/) bring together breakthrough technology to optimize and modernize observability data management offering new levels of performance and platform independence.
+
+The challenges of security and log management are only partly solved by existing software solutions. HPE and Cribl address the remaining problems of optimizing, routing, and replaying logs to provide independence from the industry’s software products in this space. HPE provides a robust way to run multiple log management software solutions and the Cribl Stream in a modern, easy-to-use, and robust platform. Together HPE and Cribl reduce the total cost of ownership of log management systems by optimizing the software, accelerating infrastructure, and reducing management costs.
+
+Cribl Stream is a vendor-agnostic observability pipeline that gives you the flexibility to collect, reduce, enrich, normalize, and route data from any source to any destination within your existing data infrastructure. HPE GreenLake is a private and hybrid cloud service that delivers the benefits of public cloud to your on-premises environment.
+
+Deploying Cribl Stream containers on HPE GreenLake offers a number of advantages, including:
+
+* **Agility:** Cribl Stream containers can be deployed quickly and easily on HPE GreenLake, giving you the agility to scale your observability pipeline up or down as needed.
+* **Cost savings:** Cribl Stream containers can help you reduce the cost of your observability pipeline by optimizing your data storage and processing through data reduction, data normalization and log routing.
+* **Security:** Cribl Stream containers can help you secure your data by encrypting it at rest and in transit.
+* **Management simplicity:** HPE GreenLake provides a single management console for managing your Cribl Stream containers, making it easy to keep your observability pipeline running smoothly.
+
+![Cribl architecture diagram](/img/criblarchitecure.png "Cribl architecture")
```

Comment:
In the architecture diagram, I do not see HPE GreenLake in the picture.

Reply:
The image seems a little fuzzy to me. Is there any way you could get a more clear picture? Also, I agree with Denis... You don't show HPE GreenLake in the picture. Seeing this, I (personally) understand that this is really about how Cribl Stream works with the HPE Ezmeral Data Fabric. But you have to remember that a lot of our customers really don't understand what's under the hood of HPE GreenLake. You might want to drop in some mention in the blog post about how HPE Ezmeral Runtime Enterprise and Data Fabric work to supply HPE GreenLake with certain functionality. It could be as easy as saying "HPE GreenLake gives you the ability to do xyz through its integrated HPE Ezmeral software technologies. Cribl Stream takes advantage of this to do such-and-such..."

Does that make sense for you?